packages/core/src/utils/getRandomNodeId.ts

```diff
@@ -1,3 +1,3 @@
-import shortid from 'shortid';
+import { nanoid } from 'nanoid';
 
-export const getRandomNodeId = shortid;
+export const getRandomNodeId = nanoid;
```

Comment:
Would it be possible to get a shorter id? 

https://github.com/ai/nanoid/#custom-alphabet-or-size

Reply:
I shortened the id from 21 to 10. 7 (like shortid used - felt waaaay to short).